cmd/nodecmd/deploy.go

```diff
@@ -33,6 +34,7 @@ It saves the deploy info both locally and remotely.
 	}
 	cmd.Flags().BoolVar(&subnetOnly, "subnet-only", false, "only create a subnet")
 	cmd.Flags().BoolVar(&avoidChecks, "no-checks", false, "do not check for healthy status or rpc compatibility of nodes against subnet")
+	cmd.Flags().StringVar(&subnetAlias, "subnet-alias", "", "subnet alias to be used for RPC calls. defaults to subnet blockchain ID")
```

Comment:
lets just use the blockchain name as alias

Reply:
k